apps/communications/contacts/js/views/multiple_select.js

```diff
@@ -0,0 +1,136 @@
+/* global Contacts, contacts, utils, LazyLoader */
+'use strict';
+
+window.Contacts = window.Contacts || {};
+
+Contacts.MultipleSelect = (function() {
+  const STATUS_TIMEOUT = 3000; // ms before closing view which shows status msg
+  var contactsToImport = [];
+  var header, saveButton, container, template, title;
+
+  function init() {
+    header = document.getElementById('multiple-select-view-header');
+    saveButton = document.getElementById('save-button');
+    container = document.getElementById('multiple-select-container');
+    template = document.getElementById('contactToImportTemplate');
+    title = document.getElementById('multiple-select-view-title');
+
+    saveButton.addEventListener('click', importAll);
+    header.addEventListener('action', cancelHandler);
+  }
+
+  function cancelHandler() {
+    header.removeEventListener('action', cancelHandler);
+    saveButton.removeEventListener('click', importAll);
+    Contacts.cancel();
+  }
+
+  function importAll() {
+    var numDupsMerged = 0,
+        importedContacts = 0;
+    const DEPENDENCIES = [
+      '/shared/js/contacts/import/utilities/status.js',
+      '/shared/js/simple_phone_matcher.js',
+      '/shared/js/contacts/contacts_matcher.js',
+      '/shared/js/contacts/contacts_merger.js',
+      '/shared/js/contacts/merger_adapter.js'
+    ];
+    LazyLoader.load(DEPENDENCIES, () => {
+      contactsToImport.forEach((contact, index) => {
+        contacts.Matcher.match(contact, 'passive', {
+          onmatch: (matches) => {
+            var callbacks = {
+              success: () => {
+                numDupsMerged++;
+                _doContinue(index);
```

Comment:
even if the contact has been merged it has actually been imported, thus we would need to increment the number of imported contacts as well